test/scenarios/idempotence/molecule/raises/molecule.yml

```diff
@@ -17,8 +17,9 @@ provisioner:
     destroy: ../../../../resources/playbooks/docker/destroy.yml
   lint:
     name: ansible-lint
-scenario:
-  name: raises
+# commented on purpose to validate that molecule will inherit it from parent folder name:
```

Comment:
It's not clear what this is testing. Can we see a clearer example or a unit test as well?

Reply:
Please disregard if this is not helpful, but I love this feature and want to see it make it to 2.20. To that end, I took a quick stab at a unit test:
```diff
diff --git a/test/unit/test_config.py b/test/unit/test_config.py
index 76c6abb8..ef5e2636 100644
--- a/test/unit/test_config.py
+++ b/test/unit/test_config.py
@@ -448,6 +448,13 @@ def test_interpolate_raises_on_failed_interpolation(patched_logger_critical,
     patched_logger_critical.assert_called_once_with(msg)
 
 
+def test_get_defaults(config_instance, mocker):
+    mocker.patch.object(config_instance, 'molecule_file',
+                        '/path/to/test_scenario_name/molecule.yml')
+    defaults = config_instance._get_defaults()
+    assert defaults['scenario']['name'] == 'test_scenario_name'
+
+
 def test_preflight(mocker, config_instance, patched_logger_info):
     m = mocker.patch('molecule.model.schema_v2.pre_validate')
     m.return_value = None
```